### PR TITLE
DOC: Drupal CMS Integration.

### PR DESCRIPTION
**Pull request by VincenzoGambino**
#### Description
This PR contains the steps to integrate Astro with Drupal.
It explains how to get Content from Drupal and what additional npm packages you need.
It also explains how to create a News site.

#### Related issues & labels (optional)

- Closes [Drupal CMS Integration Documentation](https://github.com/withastro/docs/issues/9491)
- Suggested label: Add New Content

I am a first-time contributor. My Discord username is Vincenzo Gambino

### CODE REVIEW

File: src/content/docs/en/guides/cms/drupal.mdx [hunk 1]
```diff
@@ -0,0 +1,540 @@
...
+
+### Prerequisites
+
+To get started, you will need to have the following:
+
+1. **An Astro project** - If you don't have an Astro project yet, our [Installation guide](/en/install-and-setup/) will get you up and running in no time.
+
+2. **A Drupal site** - If you haven't set up a Drupal site, you can follow the official guidelines [Installing Drupal](https://www.drupal.org/docs/getting-started/installing-drupal). Alternatively you can:
+    - **Pantheon**: Set up your application using the [Pantheon](https://pantheon.io/) free plan.
+    - **DDEV**: Set up your application locally using [DDEV](https://ddev.readthedocs.io/) by following the [Drupal CSM Quickstart](https://ddev.readthedocs.io/en/latest/users/quickstart/#drupal)
```
Comment (PierrePaul) [inline src/content/docs/en/guides/cms/drupal.mdx:25]: Drupal CSM -> Drupal CMS

**Comment by sarah11918** [comment]
Thanks so much for this contribution @VincenzoGambino ! I'm looking forward to taking a look at it, and sorry that it's been a bit busy. Just wanted to make sure I commented to let you know it's on the stack! 💜

**Comment by sarah11918** [comment]
Hi @VincenzoGambino ! I went over this today and this is fantastic!  :tada:

I did some polishing (used the `<Steps>` component, edited for Astro docs style and language etc.) so please see what you think, and correct any mistakes I might have made or things I got wrong!

I think this will be a wonderful addition to the docs, and now I think it will be very close!

**Comment by VincenzoGambino** [comment]
> Hi @VincenzoGambino ! I went over this today and this is fantastic! 🎉
> 
> I did some polishing (used the `<Steps>` component, edited for Astro docs style and language etc.) so please see what you think, and correct any mistakes I might have made or things I got wrong!
> 
> I think this will be a wonderful addition to the docs, and now I think it will be very close!

Hi Sarah,
Thank you very much.
It looks good to me!

**Review by sarah11918 — APPROVED** [review]
Thank you again @VincenzoGambino for an amazing guide! We appreciate your contribution, and are very helpful to welcome you to Team Docs! 🥳

**Comment by VincenzoGambino** [comment]
> Thank you again @VincenzoGambino for an amazing guide! We appreciate your contribution, and are very helpful to welcome you to Team Docs! 🥳

Thank you, @sarah11918 !!!!!
I am looking forward to seeing it in production!

Cheers!

**Pull request merged**